src/main/java/TaskHelper.java

```diff
@@ -0,0 +1,81 @@
+import java.util.ArrayList;
+
+public class TaskHelper {
+
+
+    public TaskHelper() {
+
+    }
```

Comment:
There is no content inside this method, maybe there was an oversight on this? Perhaps a removal of this method might reduce confusion